### DISCUSSION TITLE
When peoples quotes that are in favorites gets changed, the user is notified via notifications or a small indicator in the favorites menu.

### DISCUSSION THREAD

**Discussion by Warrobot10** [monkeytypegame/monkeytype#4683] [Ideas]
Just think that this would be helpful for people, and would be a good QOL improvement,

**Comment by Miodec** [comment]
What do you mean changed

**Reply by Warrobot10** [reply]
When a new release rolls by, normally quotes will have been changed or removed due to reports. We could be able to notify users if one of their favorite quotes are changed in the release.
